### PR TITLE
Add ImJoy integration

### PR DESCRIPTION
**Pull request by oeway**
This PR will turn elFinder into an ImJoy plugin, so you can call it directly:
Start the `imjoy-elfinder` with:
```bash
imjoy-elfinder --expose-real-path --thumbnail
```
Then in a ImJoy plugin when you want to call the plugin:
```js
    const w = await api.showDialog({type: 'external', src: 'http://127.0.0.1:8765/'})
    const selections = await w.getSelections()
    api.alert(selections[0].path)
```

Fixes https://github.com/imjoy-team/imjoy-elfinder/issues/94

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,4 +4,5 @@ pillow
 pyramid
 pyramid_jinja2
 typing_extensions
-werkzeug
\ No newline at end of file
+werkzeug
+waitress
```
Comment (MartinHjelmare) [inline requirements.txt:8]: Please sort 🔡.

Comment (oeway) [reply requirements.txt:8]: BTW, can black do automatic sorting?

Comment (MartinHjelmare) [reply requirements.txt:8]: Don't think so. But vscode can do it manually for a selection.

Comment (oeway) [reply requirements.txt:8]: I will solve it in the other PR.

Comment (oeway) [reply requirements.txt:8]: > Don't think so. But vscode can do it manually for a selection.

Nice, I get it now: https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines

Comment (MartinHjelmare) [reply requirements.txt:8]: I think there's native support too in vscode.

Comment (oeway) [reply requirements.txt:8]: Oh, really, how do you use/enable that?

**Comment by oeway** [comment]
It works like a charm!

File: requirements.txt (hunk 1 above)
Comment (MartinHjelmare) [reply requirements.txt]: I didn't activate anything that I know of. Just select a range of lines to sort, then activate the command palette and type `sort lines ascending`. You'll see the command.

Comment (oeway) [reply requirements.txt]: I had to install that extension, otherwise I don't see such option.

**Pull request merged**